Makefile.js

```diff
@@ -208,6 +208,36 @@ function getFirstVersionOfFile(filePath) {
     return getTagOfFirstOccurrence(filePath);
 }
 
+/**
+ * Gets the commit that deleted a file.
+ * @param {string} filePath The path to the deleted file.
+ * @returns {string} The commit sha.
+ */
+function getCommitDeletingFile(filePath) {
+    var commits = execSilent("git rev-list HEAD -- " + filePath);
```

Comment:
Can you add caching to this the same way I did for first version? This tasks take a while to run.